common/config/rush/version-policies.json

```diff
@@ -5,8 +5,9 @@
   },
   {
     "policyName": "itwin-tree-widget",
-    "definitionName": "individualVersion",
-    "lockedMajor": 0
+    "definitionName": "lockStepVersion",
+    "version": "1.0.0-dev",
```

Comment:
```suggestion
    "version": "1.0.0-dev.0",
```

Reply:
i think you might need to update the version in the pkg.json too

Reply:
Setting version to `1.0.0-dev.0` results in `rush version --bump` bumping `tree-widget-react` version to `1.0.0-dev.1`. Current configuration bumps it to `1.0.0-dev.0`.

As I understand from rush docs if lockstep policy is used it does not look at version in `package.json` and only uses `version` property from `version-policy`. https://rushjs.io/pages/configs/version-policies_json/